### DISCUSSION TITLE
Segmentation results for 2d ultrasound images are null

### DISCUSSION THREAD

**Discussion by sumaira-hussain** [MIC-DKFZ/nnUNet#871] [Q&A]
I have used nnUNet to train on my image dataset of 2D ultrasound images, that have been converted to nifti images following the instructions and code given. After training the prediction generated by the network have no segments inside the segmented images. I cannot understand where is the problem. Can you guide what is wrong here?

**Comment by FabianIsensee** [comment]
solved here https://github.com/MIC-DKFZ/nnUNet/issues/872
